theorems/T000641.md

```diff
@@ -0,0 +1,14 @@
+---
+uid: T000641
+if:
+  and:
+  - P000001: true
+  - P000202: true
```

Comment:
```suggestion
  - P000202: true
  - P000001: true
```
Putting P202 first in the list of hypotheses, as it's the more important property of the two (looks better in the list of theorems for P202).